### PR TITLE
CORDA-1817: Fix issue with misleading capsule error messages

### PR DESCRIPTION
**Pull request by anthonykeenan**
If there are errors creating the cordapp directory, rather than throw an error when modifying the classpath, just return the existing classpath. When the main method is invoked in the `Corda.jar` it will check again and fail gracefully.

Also bring capsule in line with the options accepted by the main Corda jar.

### CODE REVIEW

File: node/build.gradle [hunk 1]
```diff
@@ -151,6 +151,7 @@ dependencies {
     // Capsule is a library for building independently executable fat JARs.
     // We only need this dependency to compile our Caplet against.
     compileOnly "co.paralleluniverse:capsule:$capsule_version"
+    testCompile "co.paralleluniverse:capsule:$capsule_version"
```
Comment (sollecitom) [inline node/build.gradle]: I thought `compileOnly` dependencies were available within test scope as well. Am I wrong?

File: node/src/main/java/CordaCaplet.java [hunk 2]
```diff
@@ -152,17 +181,20 @@ private static void checkJavaVersion() {
...
                 logOnFailedCordappDir();
-                throw new RuntimeException("Cordapps dir could not be created"); // Let Capsule handle the error (log error, clean up, die).
+                //throw new RuntimeException("Cordapps dir could not be created"); // Let Capsule handle the error (log error, clean up, die).
+                return false;
             }
         }
         catch (SecurityException | NullPointerException e) {
             logOnFailedCordappDir();
-            throw e; // Let Capsule handle the error (log error, clean up, die).
+            //throw e; // Let Capsule handle the error (log error, clean up, die).
```
Comment (sollecitom) [inline node/src/main/java/CordaCaplet.java]: What about this commented line?

File: node/build.gradle (hunk 1 above)
Comment (anthonykeenan) [reply node/build.gradle]: Well it didn't compile without that dependency, so I guess so. Although I'm not sure whether there is a better way of doing it.

File: node/src/main/java/CordaCaplet.java (hunk 2 above)
Comment (anthonykeenan) [reply node/src/main/java/CordaCaplet.java]: Removed

**Pull request merged**